src/features/SendAssets/SendAssets.reducer.tsx

```diff
@@ -92,7 +92,7 @@ export const sendAssetsReducer = (state: State, action: ReducerAction): State =>
       const txReceipt = signedTx
         ? createPendingTxReceipt(state, action.payload.hash as ITxHash)
         : undefined;
-      return { ...state, txReceipt };
+      return { ...state, send: false, txReceipt };
```

Comment:
can you explain what this does?

Reply:
`send` marks that we are ready to send for signing for non web3 wallets. I just forgot to reset the value when the send was done.